package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "singularity",
+  "version": "1.0.0",
+  "license": "Apache-2.0 OR MIT",
+  "homepage": "https://data-programs.gitbook.io/singularity/overview/readme",
```

Comment:
<img width="1410" alt="Screenshot 2023-07-27 at 9 05 31 AM" src="https://github.com/data-preservation-programs/singularity/assets/12418265/af48b1e0-9288-4189-8ce0-745b9ceb71f2">

Not sure if it's just me, but this link goes to a "page not found" in Gitbook (seems like if I change the language from Chinese to English it loads properly after, with the link being https://data-programs.gitbook.io/singularity/v/english/overview/readme) 

This link is also in `README.md` so could be good to update there too

Reply:
It works now by changing the default language to English in gitbook